boa/src/builtins/string/mod.rs

```diff
@@ -230,6 +228,38 @@ impl String {
             .ok_or_else(|| context.construct_type_error("'this' is not a string"))
     }
 
+    /// `String.fromCodePoint(num1[, ...[, numN]])`
+    ///
+    /// The static `String.fromCodePoint()` method returns a string created by using the specified sequence of code points.
+    ///
+    /// More information:
+    ///  - [ECMAScript reference][spec]
+    ///  - [MDN documentation][mdn]
+    ///
+    /// [spec]: https://tc39.es/ecma262/#sec-string.fromcodepoint
+    /// [mdn]: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String/fromCodePoint
+    pub(crate) fn from_code_point(
```

Comment:
Could we add the spec steps as comments here to understand better where is the logic coming from?

Reply:
Done. I added the spec steps comments to only this method and lastIndexOf to prevent this PR grows too much.